### DISCUSSION TITLE
Colour and them not working on Macbook Air M1

### DISCUSSION THREAD

**Discussion by philipperobertt** [dracula/dracula-theme#830] [Q&A] [answered]
Dear community,

As it is possible to see on the attached files, the theme of Dracula (file #2) is not working on my iTerm terminal (file #1). 

I followed all the simple step on Dracula's website (https://draculatheme.com/iterm), but it does not work. 
It is worth nothing that I am new in the field. 

I am looking forward for some help 🧛

Thank you !!
<img width="567" alt="Screenshot 2023-03-16 at 11 05 37 PM" src="https://user-images.githubusercontent.com/107368211/225763277-00522e2a-4032-4a94-82e3-6fb9acd2e629.png">
![Capture-2023-03-16-225928](https://user-images.githubusercontent.com/107368211/225763479-33a53d73-9fd4-4ec2-80e5-071f883ba61d.png)

**Comment by benknoble** [comment]
If you run no commands with color output, you won't see the theme colors. The colorscheme for iTerm only changes the terminal emulator's color palette. Try some `git` commands, or `ls -G` (macOS), or add some colors to your prompt variable using the xterm escape sequences.

(The example shot is kind of a poor example for expectations because it shows an Oh-My-Zsh theme which uses colors, and a plugin for the same which highlights command lines with some syntax.)

**Reply by benknoble** [reply, marked as answer]
To add more details on prompts: I don't use Oh-My-Zsh and cannot recommend it or how to configure it.

You can see an example of a custom ZSH prompt [here in my dotfiles](https://github.com/benknoble/Dotfiles/blob/3249e0752a547e84e1da610f3a2b12a2e9e1c846/links/zshrc#L25-L89).
